mobile-app/pubspec.yaml

```
@@ -5,69 +5,69 @@ version: 5.0.3+50003
 environment:
   sdk: ">=3.0.0 <4.0.0"
 dependencies:
-  algolia_helper_flutter: ^1.2.0
-  audio_service: ^0.18.18
-  auth0_flutter: ^1.9.0
-  cached_network_image: ^3.4.1
-  device_info_plus: ^11.3.0
-  dio: ^5.8.0+1
-  firebase_analytics: ^11.4.5
-  firebase_core: ^3.13.0
-  firebase_crashlytics: ^4.3.5
-  firebase_remote_config: ^5.4.3
+  algolia_helper_flutter: 1.2.0
+  audio_service: 0.18.18
+  auth0_flutter: 1.9.0
+  cached_network_image: 3.4.1
+  device_info_plus: 11.3.0
+  dio: 5.8.0+1
+  firebase_analytics: 11.4.5
+  firebase_core: 3.13.0
+  firebase_crashlytics: 4.3.5
+  firebase_remote_config: 5.4.3
   flutter:
     sdk: flutter
-  flutter_custom_tabs: ^2.4.0
-  flutter_dotenv: ^5.2.1
-  flutter_heatmap_calendar: ^1.0.5
-  flutter_highlight: ^0.7.0
-  flutter_hooks: ^0.21.2
-  flutter_html: ^3.0.0
-  flutter_html_table: ^3.0.0
-  flutter_inappwebview: ^6.1.5
-  flutter_local_notifications: ^19.1.0
+  flutter_custom_tabs: 2.4.0
+  flutter_dotenv: 5.2.1
+  flutter_heatmap_calendar: 1.0.5
+  flutter_highlight: 0.7.0
+  flutter_hooks: 0.21.2
+  flutter_html: 3.0.0
+  flutter_html_table: 3.0.0
+  flutter_inappwebview: 6.1.5
+  flutter_local_notifications: 19.1.0
   flutter_localizations:
     sdk: flutter
-  flutter_scroll_shadow: ^1.2.6
-  flutter_secure_storage: ^9.2.4
-  graphql: ^5.2.0
-  html: ^0.15.5+1
-  infinite_scroll_pagination: ^5.0.0
-  jiffy: ^6.3.2
-  just_audio: ^0.10.0
-  path: ^1.9.1
-  path_provider: ^2.1.5
-  phone_ide: ^1.5.1
-  photo_view: ^0.15.0
-  pretty_dio_logger: ^1.4.0
-  quick_actions: ^1.1.0
-  share_plus: ^10.1.4 # NOTE: v11 has breaking changes
-  shared_preferences: ^2.5.3 # TODO: update deprecated code
-  sqflite: ^2.4.2
+  flutter_scroll_shadow: 1.2.6
+  flutter_secure_storage: 9.2.4
+  graphql: 5.2.0
+  html: 0.15.5+1
+  infinite_scroll_pagination: 5.0.0
+  jiffy: 6.3.2
+  just_audio: 0.10.0
+  path: 1.9.1
+  path_provider: 2.1.5
+  phone_ide: 1.5.1
+  photo_view: 0.15.0
+  pretty_dio_logger: 1.4.0
+  quick_actions: 1.1.0
+  share_plus: 10.1.4 # NOTE: v11 has breaking changes
+  shared_preferences: 2.5.3 # TODO: update deprecated code
+  sqflite: 2.4.2
   # TODO: Replace with sqflite methods as below package isn't actively maintained
-  sqflite_migration_service: ^2.0.0-nullsafety.1
-  stacked: ^3.4.3 # NOTE: 3.4.4+ is blocked by sqflite_migration_service
-  stacked_services: ^1.6.0
-  ua_client_hints: ^1.4.1
-  upgrader: ^11.3.1
-  url_launcher: ^6.3.1
-  web_socket_channel: ^3.0.3
-  youtube_player_iframe: ^5.1.2
+  sqflite_migration_service: 2.0.0-nullsafety.1
+  stacked: 3.4.3 # NOTE: 3.4.4+ is blocked by sqflite_migration_service
+  stacked_services: 1.6.0
+  ua_client_hints: 1.4.1
+  upgrader: 11.3.1
+  url_launcher: 6.3.1
+  web_socket_channel: 3.0.3
+  youtube_player_iframe: 5.1.2
```

```suggestion
  youtube_player_iframe: 5.2.1
```

Only this package got downgraded from the lockfile so we can set it to the lockfile one since we've been using that version for some time now